node/ltnodeclient.js

```diff
@@ -93,10 +93,12 @@ function sbRequire(require, resolve, path) {
   if(!getSB(path)) {
     if(path.match(/[\\\/]/) && fpath.extname(path) == ".js") {
       var sb = sandboxes[path.toLowerCase()] = context(path);
```

Comment:
A number of people desire a better JavaScript experience with Light Table and I think it would be great to have an active maintainer for this plugin. You also seem to have a long history of contributing to open source, which is beneficial.

My main concern is if you are familiar, or at least willing to learn, Clojure or ClojureScript? Both Light Table and some of this plugin are written in them. Aside from this, I don't have any qualms... @rundis , @cldwalker - any concerns or thoughts?

This pull request is likely not the proper location to have this discussion, but perhaps our [Gitter](https://gitter.im/LightTable/LightTable) would be better.